lua/hawtkeys/init.lua

```diff
@@ -1,25 +1,69 @@
 local M = {}
+
+---@alias SupportedKeyboardLayouts "qwerty" | "dvorak"
+
+---@class HawtKeyConfig
+---@field leader string
+---@field homerow number
+---@field powerFingers number[]
+---@field keyboardLayout SupportedKeyboardLayouts
+---@field customMaps { [string] : TSKeyMapArgs | WhichKeyMapargs } | nil
+
+---@class HawtKeyPartialConfig
+---@field leader string | nil
+---@field homerow number | nil
+---@field powerFingers number[] | nil
+---@field keyboardLayout SupportedKeyboardLayouts | nil
+---@field customMaps { [string] : TSKeyMapArgs | WhichKeyMapargs } | nil
+---
+
+---@type { [string] : TSKeyMapArgs | WhichKeyMapargs }---
+
+M._defaultSet = {
```

Comment:
Should this be a local? Seems like it shouldn't need to be exposed externally other than through config